File: java/sprint5/D/Solution.java
public class Solution {
    public static boolean treeSolution(Node head1, Node head2) {
        // Your code
        // “ヽ(´▽｀)ノ”
    }

    /** Comment it before submitting 
    private static class Node {
        int value;  
        Node left;  
        Node right;  
    
        Node(int value) {  
            this.value = value;
            this.left = null;
            this.right = null;
        }

        Node(int value, Node left, Node right) {  
            this.value = value;
            this.left = left;
            this.right = right;
        }
    }
    **/
    
    private static void test() {
        Node node1 = new Node(1,  null,  null);
        Node node2 = new Node(2,  null,  null);
        Node node3 = new Node(3,  node1,  node2);
        Node node4 = new Node(1,  null,  null);
        Node node5 = new Node(2,  null,  null);
        Node node6 = new Node(3,  node4,  node5);
        assert treeSolution(node3, node6);
    }
}

File: swift/sprint5/D/solution.swift
/*
 Comment it before submitting

 open class Node {
     
     public var value: Int
     public var left: Node?
     public var right: Node?
     
     public init(_ value: Int) {
         self.value = value
         self.right = nil
         self.left = nil
     }
 }
 */

func solution(node1: Node?, node2: Node?) -> Bool {
    // Your code
    // ヽ(´▽`)/
    return false
}


func test() {
    var node1 = Node(1)
    var node2 = Node(2)
    var node3 = Node(3)
    node3.left = node1
    node3.right = node2
    var node4 = Node(1)
    var node5 = Node(2)
    var node6 = Node(3)
    node6.left = node4
    node6.right = node5
    assert(solution(node1: node3, node2: node6))
}
